sda-orch/templates/_helpers.yaml

```diff
@@ -0,0 +1,58 @@
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "sda.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "sda.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "sda.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "tlsPath" -}}
+{{- ternary "/tls/certs" .Values.tlsPath (empty .Values.tlsPath) -}}
+{{- end -}}
+
+
+{{/**/}}
+{{- define "dbOutUser" -}}
+{{- ternary "lega_out" .Values.db.userOutgest (empty .Values.db.userOutgest) -}}
```

Comment:
I know e.g. sda-svc writes these in the same way, but is there a reason for not just doing `default "lega_out" .Values.db.userOutgest`?

Reply:
true noticed that, will simplify it. I though i would need more complex checks